README.md

```diff
@@ -39,9 +39,13 @@ npm run storybook
 ```
 Execute the following script to start e2e tesing
 ```sh
-npm run cypress:open
+npm run test:e2e
```

Comment:
we need to be inside storybook folder to run this command?, you might want to include that too. 
This will be:
```
cd packages/storybook
npm run test:e2e
```

Reply:
I had added it to the step above it. Still I'll add it to this step too as it'll be more clear.